### PR TITLE
[GameDB] Fixes, serial and patches update.

### PR DESCRIPTION
**Pull request by atomic83GitHub**
This PR add:
- A Emotion Engine timing fix for World super police (Tested by @prafullpcsx2 and Discord member)
- The rounding fix for the international version of Final Fantasy X (Tested by @Mensa41 Close https://github.com/PCSX2/pcsx2/issues/2891)
- A Emotion Engine Clamping fix for let's make a soccer team (Tested by @atomic83GitHub)
- A Emotion Engine rounding fix for Eternal Ring (Tested by @atomic83GitHub)
- Add the WRC avec Sebastien Loeb patch. (Made by @prafullpcsx2, tested by @atomic83GitHub)

There are also several additons about Winback fixes for various regions.

### CODE REVIEW

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -139,10 +139,10 @@ Name   = Playstation 2 - Demo Disc 2001
 Region = PAL-M5
 [patches]
 	// 7ACF7E03
-	comment=You must enable the EETimingHack when playing the WRC demo to avoids random hangs.
-	comment=You must change the EE Clamping mode to Full when playing the Klonoa 2 demo to avoids misplaced objects.
-	comment=You must change the VU Clamping mode to extra when playing the Klonoa 2 demo to fixes reflections issues.
 	comment=You must use the ZeroSPU2 plugin when playing the Klonoa 2 demo to avoids bad/missing sounds.
+EETimingHack = 1 // Fixes WRC demo hanging when the car cross the starting line.
+eeClampMode = 3 // In the Klonoa 2 demo objects appear in wrong places without it.
+vuClampMode = 2 // Fixes water reflection in the Klonoa 2 demo.
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: I don't think it's a good idea to add these for demo discs, other demo games might break.

Comment (atomic83GitHub) [reply bin/GameIndex.dbf]: I tested each of them before doing this and none of them actualy break with these fixes.
But it's up to you, do you want I remove them?

Comment (lightningterror) [reply bin/GameIndex.dbf]: Yes change them back.

File: bin/GameIndex.dbf [hunk 2]
```diff
@@ -19148,6 +19164,7 @@ Region = NTSC-J
 Serial = SLPM-62023
 Name   = Winback
 Region = NTSC-J
+vuClampMode = 3 // For full textures showing.
```
Comment (MrCK1) [inline bin/GameIndex.dbf]: Could you clarify these comments a bit more for others in the future? This isn't exactly clear what it fixes ;)

Comment (atomic83GitHub) [reply bin/GameIndex.dbf]: Is it fine now?

File: bin/GameIndex.dbf (hunk 1 above)
Comment (atomic83GitHub) [reply bin/GameIndex.dbf]: Done.

File: bin/GameIndex.dbf (hunk 2 above)
Comment (lightningterror) [reply bin/GameIndex.dbf]: Looks fine to me.

**Pull request merged**